diem-move/diem-framework/experimental/demo-cli/src/request_utils.rs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::{bail, format_err, Result};
+
+use serde::{Deserialize, Serialize};
+use std::str::FromStr;
+
+const DIEM_ACCOUNT_RESOURCE: &str = "0x1::DiemAccount::DiemAccount";
+
+#[derive(Clone, Serialize, Deserialize, Debug, PartialEq)]
+pub struct AccountResource {
+    #[serde(rename(serialize = "type", deserialize = "type"))]
+    pub type_: String,
+    pub value: serde_json::Value,
```

Comment:
Heads up, I'm landing the pr changes this field name to 'data', all others are same